app/modules/auth/tests/test_models.py

```
@@ -0,0 +1,379 @@
+        provider = db_session.query(UserAuthProvider).filter_by(
+            user_id=user_id
+        ).first()
+        assert provider is None
+
+    def test_oauth_tokens(self, db_session, test_user):
+        """Test storing OAuth tokens"""
+        provider = UserAuthProvider(
+            user_id=test_user.uid,
+            provider_type="sso_google",
+            provider_uid="google-123",
+            access_token="access-token-123",
```

Ruff (0.14.8)

Possible hardcoded password assigned to argument: "access_token"

(S106)